CombinedConversionCalculation.java

```diff
@@ -0,0 +1,341 @@
+/***
+ * Question :- Conversions of all from base to others and Arithmetic operations
+ * Date :- 26/09/2024
+ * Owner :- Dikshant Goswami
+ */
+
+import java.util.Scanner;
+
+public class CombinedConversionCalculation {
+
+    // Power functions is used where the base and the power which defines how much time a value will multiply itself passes as  parameters
+    public static int powerfunction(int base, int exponent) {
+        int result = 1;
+        for (int start = 0; start < exponent; start++) {
+            result = result * base;
+        }
+        return result;
+    }
+
+    // Binary to Decimal conversion the binary number entered from user is passed as a parameter and return as the decimal
+    public static int forDecimal(String binary) {
+        int decimal = 0;
+        for (int endoflength = binary.length() - 1, powervalue = 0; endoflength >= 0; endoflength--, powervalue++) {
+            int bit = binary.charAt(endoflength) - '0';
+            if (bit == 1) {
+                decimal += powerfunction(2, powervalue);
+            }
+        }
+        return decimal;
+    }
+
+    // Decimal to Binary conversion

```

Comment:
you can use multiline comments in which you can mention short description and return parameter